tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam_nearest-projection/nearest-projection-setup.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+    <log>
+        <sink type="stream" output="stdout"  filter= "(%Severity% > debug) or (%Severity% >= trace and %Module% contains SolverInterfaceImpl)"  enabled="true" />	
+    </log>
+
+    <solver-interface dimensions="3">
+
+    <data:scalar name="Temperature"/>
+    <data:scalar name="Heat-Flux"/>
+
+    <mesh name="Fluid-Mesh-Centers">
+        <use-data name="Heat-Flux"/>
+   </mesh>
+
+   <mesh name="Fluid-Mesh-Nodes">
+        <use-data name="Temperature"/>
+  </mesh>
+
+    <mesh name="Solid-Mesh-Centers">
+        <use-data name="Temperature"/>
+    </mesh>
+
+    <mesh name="Solid-Mesh-Nodes">
+        <use-data name="Heat-Flux"/>
+    </mesh>
+
+
+    <participant name="Fluid">
+        <use-mesh name="Fluid-Mesh-Centers" provide="yes"/>
+        <use-mesh name="Fluid-Mesh-Nodes" provide="yes"/>
+        <read-data name="Heat-Flux" mesh="Fluid-Mesh-Centers"/>
+	<write-data name="Temperature" mesh="Fluid-Mesh-Nodes"/>
+    </participant>
+
+    <participant name="Solid">
+        <use-mesh name="Fluid-Mesh-Centers" from="Fluid"/>
+        <use-mesh name="Fluid-Mesh-Nodes" from="Fluid"/>
+        <use-mesh name="Solid-Mesh-Nodes" provide="yes"/>
+        <use-mesh name="Solid-Mesh-Centers" provide="yes"/>
+        <read-data name="Temperature" mesh="Solid-Mesh-Centers"/>
+        <write-data name="Heat-Flux" mesh="Solid-Mesh-Nodes"/>
+	<!--export:vtk directory="preCICE-output" /-->
```

Comment:
Leave this here, but better write it as:
```xml
<!-- <export:vtk directory="preCICE-output" /> -->
```
so that, when uncommenting, the correct tag will remain.

```suggestion
	<!-- <export:vtk directory="preCICE-output" /> -->
```